api/service/api.go

```diff
@@ -217,18 +217,18 @@ func (a *API) blobSidecarHandler(w http.ResponseWriter, r *http.Request) {
 
 // filterBlobs filters the blobs based on the indices query provided.
 // If no indices are provided, all blobs are returned. If invalid indices are provided, an error is returned.
-func filterBlobs(blobs []*deneb.BlobSidecar, indices string) ([]*deneb.BlobSidecar, *httpError) {
-	if indices == "" {
-		return blobs, nil
-	}
-
-	splits := strings.Split(indices, ",")
-	if len(splits) == 0 {
+func filterBlobs(blobs []*deneb.BlobSidecar, _indices []string) ([]*deneb.BlobSidecar, *httpError) {
+	var indices []string
+	if len(_indices) == 0 {
 		return blobs, nil
+	} else if len(_indices) == 1 {
+		indices = strings.Split(_indices[0], ",")
+	} else {
+		indices = _indices
 	}
```

Comment:
nit: there's an edge case where someone does:

```
?indices=0&indices=1,2
```

But that's probably overkill to solve for.